docs/pages/project/contributing/contributing-ui-tests.md

```diff
@@ -9,40 +9,121 @@ category: contributing
 list: include
 ---
 
-To automate functional integration and end-to-end testing Meshery uses [Playwright](https://playwright.dev) as one of the tools to automate browser testing. End-to-end tests run with each pull request to ensure that the changes do not break the existing functionality.
+To automate functional integration and end-to-end testing Meshery uses [Playwright](https://playwright.dev/) as one of the tools to automate browser testing. End-to-end tests run with each pull request to ensure that the changes do not break the existing functionality.
 
-## Executing Tests
+## Prerequisites:
 
-Clone the `meshery/meshery` repo. Tests are written in JavaScript. Navigate to the [/ui/tests/e2e/](https://github.com/meshery/meshery/tree/master/ui/tests/e2e) directory.
+Before diving into Meshery's testing environment, certain prerequisites are necessary:
 
-Install the dependencies by running the following command:
+- A verified Layer5 cloud account.
```

Comment:
We should be able to support use of an API token from the Local Provider as well, right?

Reply:
How about this?
![image](https://github.com/user-attachments/assets/080f9aee-a9b0-498c-8530-a4c384800e18)